test/functional/test-extensions.js

```diff
@@ -25,52 +25,31 @@ import {
   installExtensionsService,
   registerExtension,
 } from '../../src/service/extensions-impl';
-import {adopt} from '../../src/runtime';
-import {
-  createIframePromise,
-  doNotLoadExternalResourcesInTest,
-} from '../../testing/iframe';
 import {
   initLogConstructor,
   resetLogConstructorForTesting,
 } from '../../src/log';
 import {loadPromise} from '../../src/event-helper';
-import * as cust from '../../src/custom-element';
-import * as sinon from 'sinon';
-
+import * as describes from '../../testing/describes';
```

Comment:
I also made `sandbox` global. Just `sandbox` for now to see how it plays out. We may do this for other vars as well.